chain/jsonrpc/src/lib.rs

```diff
@@ -1461,6 +1462,33 @@ async fn epoch_info_html() -> actix_web::Result<impl actix_web::Responder> {
     Ok(HttpResponse::Ok().body(*EPOCH_INFO_HTML))
 }
 
+#[derive(Deserialize, Debug)]
+struct LogLevelInfo {
+    rust_log: String,
+}
+
+#[derive(Deserialize, Debug)]
+struct LogLevelVerboseInfo {
+    rust_log: Option<String>,
+    verbose_module: Option<String>,
+}
+
+#[get("/debug/log_level")]
+async fn log_level(info: web::Query<LogLevelInfo>) -> actix_web::Result<impl actix_web::Responder> {
+    let result = reload_env_filter(&Some(&info.rust_log), false, &None)
+        .map_or_else(|err| format!("Failed to set env filter: {}", err), |_| "OK".to_string());
+    Ok(HttpResponse::Ok().json(result))
+}
+
+#[get("/debug/log_level_verbose")]
+async fn log_level_verbose(
+    info: web::Query<LogLevelVerboseInfo>,
+) -> actix_web::Result<impl actix_web::Responder> {
+    let result = reload_env_filter(&info.rust_log.as_ref(), true, &info.verbose_module)
+        .map_or_else(|err| format!("Failed to set env filter: {}", err), |_| "OK".to_string());
+    Ok(HttpResponse::Ok().json(result))
+}
+
```

Comment:
I’d rather we kill --verbose flag to be honest and so not include other interfaces for setting it.